staging/scripts/deploy-latest-to-support

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+echo "Running deploy latest to support"
+
+SUPPORTHOST=$(ssh-keyscan support.montagu.dide.ic.ac.uk)
```

Comment:
I don't think we should do this here, but instead on machine provision. Can you remove it here, run manually on the 3 agents for now, and add as a change in the montagu-ci repo? Then the logic can go away